conf/app.routes

```diff
@@ -1,9 +1,10 @@
 # microservice specific routes
 
 ## Non-Resident Calculation Routes
-GET        /calculate-flat              controllers.nonresident.CalculatorController.calculateFlat(model: CalculationRequestModel)
-GET        /calculate-rebased           controllers.nonresident.CalculatorController.calculateRebased(model: CalculationRequestModel)
-GET        /calculate-time-apportioned  controllers.nonresident.CalculatorController.calculateTA(model: TimeApportionmentCalculationRequestModel)
+GET        /calculate-flat                          controllers.nonresident.CalculatorController.calculateFlat(model: CalculationRequestModel)
+GET        /calculate-rebased                       controllers.nonresident.CalculatorController.calculateRebased(model: CalculationRequestModel)
+GET        /calculate-time-apportioned              controllers.nonresident.CalculatorController.calculateTA(model: TimeApportionmentCalculationRequestModel)
+GET        /non-resident/calculate-total-gain       controllers.nonresident.CalculatorController.calculateTotalGain(disposalValue: Double, disposalCosts: Double, acquisitionValue: Double, acquisitionCosts: Double, improvements: Double, rebasedValue: Option[Double], rebasedCosts: Option[Double], disposalDate: Option[org.joda.time.DateTime], acqusitionDate: Option[org.joda.time.DateTime], improvementsAfterTaxStarted: Option[Double])
```

Comment:
we could potentially default `improvementsAfterTaxStarted` to 0 and it wouldn't have to be optional